monitoring_test.go

```diff
@@ -86,8 +86,8 @@ func TestUnmarshalMonitoringJobs(t *testing.T) {
 		t.Error("Status is not up")
 	}
 	r := j.Rules[0]
-	assert.Equal(t, r.Key, "rtt", "RTT rule key is wrong")
-	assert.Equal(t, r.Value.(float64), float64(100), "RTT rule value is wrong")
+	// assert.Equal(t, r.Key, "rtt", "RTT rule key is wrong")
```

Comment:
@bobtfish I see what you mean, that is a dope testing package the more I look into it. I may have gone a bit overboard on no external deps(this is the only one, and like you said, not in the binary). Lets keep it.